scripts/scripts.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable import/no-cycle */
 import { events } from '@dropins/tools/event-bus.js';
-import { getCartDataFromCache } from '@dropins/storefront-cart/api.js';
```

Comment:
@herzog31 note: I _will_ need to add that dynamic import to prex add to cart once this PR is merged ;)